documentation/messages/user.registration.transactional.md

```diff
@@ -0,0 +1,121 @@
+# User registration transactional message
+
+User registration event.
+
+**Routing key**
+
+`user.registration.transactional`
+
+**Exchanges**
+- `transactionalExchange`
+
+**Queues**
+- `activityStatsQueue`
+- `loggingQueue`
+- `mobileCommonsQueue`
```

Comment:
For this queue, list the second binding of `user.registration.*` rather than the first `campaign.signup.*`.